std/internal/math/biguintcore.d

```diff
@@ -2224,10 +2251,12 @@ bool inplaceSub(BigDigit[] result, const(BigDigit)[] x, const(BigDigit)[] y)
 
 /* Determine how much space is required for the temporaries
  * when performing a Karatsuba multiplication.
+ * TODO: determining a tight bound is non-trivial and depends on KARATSUBALIMIT, see:
+ * https://issues.dlang.org/show_bug.cgi?id=20493
```

Comment:
Ping me when you're happy with this `TODO`

Reply:
I was hoping to leave that for a future PR. The comment is there to explain that the formula is a bit arbitrary and not the result of rigorous mathematical analysis or benchmarking, but it should be good enough.